chapters/lexicalstructure.tex

```diff
@@ -235,14 +228,15 @@ \subsection{Strings}\label{strings}
 literal. For example, the following statement defines one string that
 contains (non-printable) newline characters:
 \begin{lstlisting}[language=modelica]
-assert(noEvent(length > s_small), "
+assert(noEvent(length > s_small),
+       "
```

Comment:
Wouldn't it make more sense to start the string as `"The distance bet...` on one line? There are still new-lines, but it doesn't look as contrived.

Reply:
Fixed.  Resolving.